resources.tf

```diff
@@ -106,3 +106,111 @@ resource "aws_s3_bucket_policy" "published_bucket_https_only" {
   bucket = aws_s3_bucket.published.id
   policy = data.aws_iam_policy_document.published_bucket_https_only.json
 }
+
+resource "aws_iam_role" "dataset_generator" {
```

Comment:
`s/dataset_generator/analytical_dataset_generator/` throughout please. We'll be having a `pdm_dataset_generator` one come along shortly.

Reply:
OK I thought we were making one role to rule them all.